lib/issue_templates_issues_hook.rb

```diff
@@ -6,30 +6,38 @@ class IssueTemplatesIssuesHook < Redmine::Hook::ViewListener
   
   def view_layouts_base_html_head(context = {})
     o = stylesheet_link_tag('issue_templates', :plugin => 'redmine_issue_templates')
-    if (context[:controller].class.name == 'IssuesController' and 
-      context[:controller].action_name != 'index') or 
-      (context[:controller].class.name == 'IssueTemplatesController') or
+    if (context[:controller].class.name == 'IssuesController' &&
+        context[:controller].action_name != 'index') ||
+      (context[:controller].class.name == 'IssueTemplatesController') ||
```

Comment:
__[rubocop]__
* Align the operands of a condition in an `if` statement spanning multiple lines.